dttools/src/Makefile

```diff
@@ -139,6 +140,9 @@ auth_globus.o: auth_globus.c
 auth_test: auth_test.o libdttools.a
 	$(CCTOOLS_LD) -o $@ $(CCTOOLS_INTERNAL_LDFLAGS) $(LOCAL_LDFLAGS) $^ $(LOCAL_LINKAGE) $(CCTOOLS_GLOBUS_LDFLAGS) $(CCTOOLS_EXTERNAL_LINKAGE)
 
+libforce_halt_enospc.so: libforce_halt_enospc.o
+	$(CCTOOLS_CC) -shared -ldl -fPIC $^ -o $@
```

Comment:
Sorry, I changed it before and undid the change due to other issues with the Makefile in between this last bit of commits. Will redo now!